### ISSUE TITLE
Sidebar doesn't accept taps anymore?

### ISSUE THREAD

**Issue by 9mm** [root-two/react-native-drawer#154]
Hey there,

My sidebar was working great pre-2.1.0 (the last version before < 2), but now when I click an item in the sidebar, the sidebar just closes and the original clicked item isn't triggered.

Here is my configuration:

```
  <DrawerIOS
    content={sidebarContent}
    disabled={!!this.props.modalValues.hasModals}
    ref="drawer"
    type="static"
    onOpenStart={() => {
      this._hideKeyboard();
      this._setStatusBarHidden(true);
    }}
    onOpen={() => this.setState({isDrawerOpened: true})}
    onCloseStart={() => this._setStatusBarHidden(false)}
    onClose={() => this.setState({isDrawerOpened: false})}
    openDrawerOffset={0.2}
    panOpenMask={0.2}
    captureGestures="open"
    negotiatePan={true}
    tapToClose={true}
    styles={styles.ios}
    tweenDuration={110}
    tweenEasing="easeInOutCirc">
    {content}
  </DrawerIOS>
```

Do you see anything wrong that I am doing here that might have broken?

**Comment by zelll** [comment]
`if (gestureState.moveX < 50) this.processTapGestures()`
This moveX is much bigger than 50 when I clicks right side of my phone screen. On simulator, it works as expected.

**Comment by rt2zz** [comment]
I noticed this as well. Maybe moveX threshold is needs to be bumped up, or we can also look at things like `dx` or pan duration

**Comment by 9mm** [comment]
It seems it was now fixed in 2.2.1.... can't tell if this is coincidence or what

edit: I tried all old versions and now I can't even get it to reproduce. It was definitely happening though, across multiple builds + etc. :|

**Comment by 9mm** [comment]
Now it appears tap to close doesnt work although it's set to true (same configuration as above)

**Comment by rt2zz** [comment]
Now I cannot recreate either. It seems to be sproradic, maybe related to wether chrome debugger is open? I am open to suggestions for how to resolve.

**Comment by 9mm** [comment]
I haven't ever used chrome debugger, so it's probably not that.

Is the 2nd problem I have reproducible to you though? (Where you cant tap to close)

**Comment by mvaivre** [comment]
@rt2zz Yeah, I often noticed a very different behavior when the chrome debugger is on (drawer can't open, or stuck in the middle of animation, or no animation at all). Do you have an idea why the debugger would induce that?

**Comment by rt2zz** [comment]
@mvaivre not sure, I think the rn pan responder implementation does not translate perfectly between jscore and v8. I upped the moveX threshold to 125 so that may help with the tapToClose issue.

**Comment by 9mm** [comment]
Thats awesome you guys fixed the tapping issue! I was curious if you knew why `tapToClose={true}` still doesn't close the sidebar though when tapping?

I could make a new issue if you want I just didn't want to spam you guys 👍

**Comment by rt2zz** [comment]
@9mm still trying to hunt down all of the facets of this bug. Since v2.2.2 I have not personally observed any tapToClose fails. are you seeing it still? If so is it iOS or android or simulator?

**Comment by 9mm** [comment]
I just tested in simulator (using the config I pasted above). Interestingly it does work properly on simulator, just not on my actual iPhone 6s

**Comment by savioseb** [comment]
I'm experiencing the same issue 
- Android running on phone
- iOS Simulator
- iOS App running on phone

Any updates about this?

```
<Drawer
                style={{marginBottom: 20}}
                ref="drawer"
                type="overlay"
        acceptTap={true}
                tapToClose={true}
                openDrawerOffset={0.2}
                panCloseMask={0.2}
        onClose={this.closeSidebar}
                content = {<Sidebar />}
                >
```

I did not experience this issue when I was running RN 0.24.1 with RN-drawer **2.0.0**

But since I've updated  my RN to 0.31.0 and RN-drawer to **2.2.6 I**'m facing this issue.

**Comment by Biplovkumar** [comment]
I am facing same issue in  
1.iOS Simulator
2.iOS App running on phone
